src/main/scala-2.11/markets/engines/MutableMatchingEngine.scala

```diff
@@ -38,4 +32,10 @@ trait MutableMatchingEngine extends MatchingEngine {
     _bidOrderBook.find(order => order.isInstanceOf[LimitOrderLike])
   }
 
+  /* Mutable collection of ask orders for internal use only! */
+  protected val _askOrderBook: mutable.Iterable[AskOrder]
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Field name does not match the regular expression '^[a-z][A-Za-z0-9]*$'](https://www.codacy.com/app/davidrpugh/markets-sandbox/file/1834547235/issues/source?bid=2921361&fileBranchId=3080901#l36)